data/hai/hai reveal 5 philosophers.txt

```diff
@@ -376,14 +382,14 @@ mission "Hai Reveal [B05-long] The Red Mystery"
 mission "Hai Reveal [B06-long] Tarazed Analysis"
 	landing
 	name "Seeking Aid"
-	description `Find and board a smugglers ship. Your display will identify such ships as belonging to the "Smuggler" government. Try Syndicate space or the Far North. Head to <planet> afterwards.`
+	description `Board smuggler ships to gain information. Your display will identify such ships as belonging to the "Smuggler" government. Try near Devil's Run in Syndicate space, or the Far North.`
```

Comment:
The first half, maybe, the second half, no.

Reply:
The mystery of the outcome of boarding a ship is more interesting, so I'll revert this change.